### ISSUE TITLE
Clarify `null` handling in different cases

### ISSUE THREAD

**Issue by nikku** [camunda/dmn-tck#67]
### What shall we do

Related to [this conversation](https://camunda.slack.com/archives/C01QYD808A3/p1671687820836529) we should clarify the handling of `null` in DMN FEEL.

### Open questions

* [x] https://github.com/dmn-tck/tck/issues/536 :arrow_right: https://github.com/dmn-tck/tck/issues/536#issuecomment-1385863569, https://github.com/dmn-tck/tck/pull/540
* [x] https://github.com/dmn-tck/tck/issues/537 :arrow_right: https://github.com/dmn-tck/tck/issues/537#issuecomment-1386634436, https://github.com/dmn-tck/tck/pull/541
* [x] https://github.com/dmn-tck/tck/issues/538 => Works as specified. String cohersion currently not covered by the spec. Created [a JIRA issue](https://github.com/dmn-tck/tck/issues/538#issuecomment-1396932809) with the RFP to drive this forward
* [x] https://github.com/dmn-tck/tck/issues/539

**Comment by nikku** [comment]
As discussed with @barmac he plans to bring attention to these issues in the DMN 1.5 RTF call later today.

**Comment by nikku** [comment]
I've added TCK test cases for the most important cases (filter and path expression `null` handling): https://github.com/dmn-tck/tck/issues/536#issuecomment-1396591619.

**Comment by nikku** [comment]
Will follow up with function invocation as I find the time. Let's get all aspects of `null` handling verified by the TCK :rocket:

**Comment by nikku** [comment]
With https://github.com/dmn-tck/tck/issues/539 clarified + additional tests merged this issue is closed.

**Comment by nikku** [comment]
**TLDR:** DMN FEEL works exactly as specified, with invalid conversions / operations resulting in `null` and an `errorResult` indicating that there was an undesired conversion.
